elliott

```diff
@@ -131,12 +131,21 @@ Bugzilla Bugs.
 # advisory:create
 #
 @cli.command("create", short_help="Create a new advisory")
+@click.option("--type", '-t', 'errata_type',
+              type=click.Choice(['RHBA', 'RHSA', 'RHEA']),
+              default='RHBA',
+              help="Type of Advisory to create.")
 @click.option("--kind", '-k', required=True,
               type=click.Choice(['rpm', 'image']),
-              help="Kind of Advisory to create. Affects boilerplate text.")
+              help="Kind of artifacts that will be attached to Advisory. Affects boilerplate text.")
 @click.option("--impetus", default='standard',
               type=click.Choice(elliottlib.constants.errata_valid_impetus),
               help="Impetus for the advisory creation [standard, cve, ga, test]")
+@click.option('--impact', required=False,
```

Comment:
Impact can be determined from the highest severity of the attached trackers. 

Severity / Impact
- Urgent = Critical
- High = Important
- Medium = Moderate
- Low = Low

Reply:
I made a pass at auto-determining this info on RHSA creation. I will need to figure out a way to test it with affecting current CVE trackers.